src/Markdig/Markdown.cs

```diff
@@ -153,13 +153,13 @@ public static MarkdownDocument Parse(string markdown)
         /// <param name="context">A parser context used for the parsing.</param>
         /// <returns>An AST Markdown document</returns>
         /// <exception cref="ArgumentNullException">if markdown variable is null</exception>
-        public static MarkdownDocument Parse(string markdown, MarkdownPipeline pipeline, MarkdownParserContext context = null)
+        public static MarkdownDocument Parse(string markdown, MarkdownPipeline pipeline, MarkdownParserContext context = null, bool trackTrivia = false)
```

Comment:
Actually, maybe we could add `TrackTrivia` as a property of `MarkdownPipeline`, and propagate the this bool to the processor when necessary?

Reply:
done